cns/restserver/util.go

```diff
@@ -622,22 +622,23 @@ func (service *HTTPRestService) SendNCSnapShotPeriodically(ncSnapshotIntervalInM
 	}
 }
 
-func (service *HTTPRestService) validateIpConfigRequest(ipConfigRequest cns.GetIPConfigRequest) (int, string) {
+func (service *HTTPRestService) validateIpConfigRequest(ipConfigRequest cns.GetIPConfigRequest) (*cns.KubernetesPodInfo, int, string) {
```

Comment:
I think it would be safer to return a `cns.KubernetesPodInfo` instead of a pointer